Services/LogService/ModsenOnlineStore.LogService.API/Controllers/LogController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using ModsenOnlineStore.LogService.Domain.Entities;
+using System;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace ModsenOnlineStore.LogService.API.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class LogController : ControllerBase
```

Comment:
do you really need logging as API?

Reply:
Maybe not, but it's a well-known approach